api/cosmos/feegrant/v1beta1/feegrant.pulsar.go

```diff
@@ -2730,7 +2730,7 @@ type AllowedMsgAllowance struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// allowance can be any of basic and filtered fee allowance.
+	// allowance can be any of basic and periodic fee allowance.
```

Comment:
where this changes came from?

Reply:
`make proto-gen` made these changes, it seems to match the comment in proto file: https://github.com/cosmos/cosmos-sdk/blob/master/proto/cosmos/feegrant/v1beta1/feegrant.proto#L61